docker-compose.yml

```diff
@@ -13,4 +15,16 @@ web:
 db:
   image: postgres
   ports:
-    - "5432"
\ No newline at end of file
+    - "5432"
+
+redis:
+  image: redis
+  ports:
+    - "6379:6379"
```

Comment:
One thing I'm not sure of here:

Have you tested this with just `"6379"` instead of `"6379:6379"`? I'm think the second option exposes the port on the boot2docker VM, and I don't think we need to do that for these instances, but I could be mistaken on both counts.